### PR TITLE
update with fixed code

### PR DESCRIPTION
**Pull request by hasjenL**

### CODE REVIEW

File: lib/fluent/plugin/in_rds_mysql_log.rb [hunk 1]
```diff
@@ -1,7 +1,18 @@
+require 'fluent:input'
```
Comment (mtanda) [inline lib/fluent/plugin/in_rds_mysql_log.rb:1]: I'm not sure, is this line required?
The plugin works without this line.

File: lib/fluent/plugin/in_rds_mysql_log.rb [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
 require 'myslog'
 
 class Fluent::RdsMysqlLogInput < Fluent::Input
-  Fluent::Plugin.register_input('rds_mysql_log', self)
+  Fluent::Plugin.register_input("rds_mysql_log", self)
```
Comment (mtanda) [inline lib/fluent/plugin/in_rds_mysql_log.rb:5]: Why do you change the quote from single quote to double quote?

**Comment by mtanda** [comment]
Thanks for contributing!

I comment some of the changes.
I didn't comment, but I think the file permissions should not changed.
Please revert the permission change.

File: lib/fluent/plugin/in_rds_mysql_log.rb (hunk 1 above)
Comment (hasjenL) [reply lib/fluent/plugin/in_rds_mysql_log.rb:1]: Sorry it should actually be require 'fluent/input'   Its needed that way for the latest version of fluentd.  I actually duplicated your plugin and made the changes i need and am adding more to meet the needs of my company.  So, i no longer need these changes here, if you do not want to merge the PR.  I pushed the changes up to ruby gems and plan to make quite a few more in the plugin itself. Also, added travis ci and rake testing which will be deployed in the next couple of days. I gave you credit for the original code.  great work. Again, if you want to close this PR please fee free.

Comment (mtanda) [reply lib/fluent/plugin/in_rds_mysql_log.rb:1]: Sorry for nitpick review comment...
Please don't consider, I don't want to merge your PR.
I appreciate your contribution 😄 

Honestly, I don't have enough time to maintain this plugin, and I'm mostly satisfied with the feature.

> I pushed the changes up to ruby gems and plan to make quite a few more in the plugin itself.

If you maintain this plugin, it's really great!!
I'll switch to use your plugin.

**Pull request closed without merging**